patches/0029-Fix-markTagsAllNotComputedForResources-to-recognize-.patch

```diff
@@ -21,34 +21,20 @@ index ce64074bfd..e24e53fe17 100644
  // rationale for this is that Pulumi copies tags to tags_all before it hits the TF layer, so these
  // attributes must match in schema.
  func markTagsAllNotComputedForResources(sdkV2Provider *schema.Provider) {
--	updatedResourcesMap := map[string]*schema.Resource{}
```

Comment:
I'm not sure why this and the optimize-startup patch has churn. They both edit the same code in `shim.go` so somehow the rebase got messed up. I confirmed that the actual shim.go code is not changing after applying the patches. 

I've created another PR to simplify some of these patches to hopefully avoid this.
https://github.com/pulumi/pulumi-aws/pull/5444